modules/document_repository/php/files.class.inc

```diff
@@ -381,7 +388,7 @@ class Files extends \NDB_Page
                         'version'       => $version,
                         'File_name'     => $fileName,
                         'File_size'     => $fileSize,
-                        'Data_dir'      => $fileName,
+                        'Data_dir'      => $uploadPath,
```

Comment:
@driusan @kongtiaowang we might have a serious problem on our hands here:

- Before LORIS 21.0 we were saving here the [FILEBASE](https://github.com/aces/Loris/blob/20.0-release/modules/document_repository/ajax/documentEditUpload.php#L93)  which looks like `$username/$filename`
- 21.0 and 22.0 we have been saving the FILENAME (as you can see in the before state of this PR) which is only `$filename`
- In this PR we are changing it to UPLOADPATH which is the absolute path !! `$configpath/*username/$filename`

All of these changes are done without taking into consideration what was already previously in the database which means all documents uploaded in previous versions would not be downloadable...

Reply:
@ridz1208 It doesn't really matter what's stored. In theory `Data_dir` can be deleted and we can construct the path using `$configpath/$username/$fileName`.

`Data_dir` has always seemed like a poor design choice to me and we should probably remove it.

Reply:
if thats the case it should just be removed then. where is it used if it's not in rebuilding the path to the file ?

Reply:
I don't think it is used. The doc repo code has been rewritten at least twice since i've worked here and I think it needs a lot of cleanup in general. 

I agree it should be removed. I'll make a cleanup ticket.

Reply:
```suggestion
                        'Data_dir'      => $fileName,
```